packages/graphql_flutter/test/widgets/query_test.dart

```diff
@@ -51,13 +52,17 @@ class Page extends StatefulWidget {
   final Map<String, dynamic>? variables;
   final FetchPolicy? fetchPolicy;
   final ErrorPolicy? errorPolicy;
-
-  Page({
-    Key? key,
-    this.variables,
-    this.fetchPolicy,
-    this.errorPolicy,
-  }) : super(key: key);
+  final OnQueryCompleted? onQueryCompleted;
+  final OnQueryError? onQueryError;
```

Comment:
```suggestion
  /// Add some docs on when it is used
  final OnQueryCompleted? onQueryCompleted;
  final OnQueryError? onQueryError;
```

I know this is not common in the library, but we start to integrate comments over a files, this make easy jump in the project as new contributor that have 0 knowledge of the library!